src/filters/EntropyCalculator.js

```diff
@@ -217,12 +218,30 @@ function convolute( dataIn){
         filterData[iCol] = smallFilter[ic];
     }
 
-    FFT.init(nbPoints)
+    FFT.init(nbPoints);
     var ftFilterData = FFT.fft(filterData);
 
     filterData = null;
     Convolution.convolute(ftSpectrum, ftFilterData);
     var convolutedSpectrum = FFT.ifft(ftSpectrum); //@TODO mirar que pasa ak
 
     return convolutedSpectrum;
+}
+
+function RealFFT(data){
+    var
+        nbPoints = data.length,
+        re = new Array(nbPoints/=2-1),
```

Comment:
nPoints/=2-1 == nPoints. You should better assign nbPoints = data.length/2-1, if it is what you want.